### PR TITLE
Add documentation for installation of Chainer with GPU

### PR DESCRIPTION
**Pull request by wkentaro**
cc @yuki-asano

I hope this **automatically** resolves problems of new users of Chainer, Cupy for Deep Learning,  
without help, power and time of @k-okada, @wkentaro and @knorth55 for example.

### CODE REVIEW

**Comment by wkentaro** [comment]
@yuki-asano ここにインストール方法などを書いておくので、jsk_tendon_robotにあるdocumentationの方で参照していただけるといいかと思います。

File: doc/install_chainer_gpu.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- Nvidia GPU (ex. K80, TitanX, GTX 1080Ti).
+- Ubuntu (ex. 14.04, 16.04).
+
+
+Install CUDA
+------------
+
+- Download deb file from https://developer.nvidia.com/cuda-downloads::
+
+    # If you'd like to use CUDA8.0 on aUbuntu 14.04.
```
Comment (knorth55) [inline doc/install_chainer_gpu.rst]: `aUbuntu` -> `Ubuntu`

**Comment by wkentaro** [comment]
@yuki-asano Could you review this?

**Comment by wkentaro** [comment]
@k-okada Could you merge this?

**Comment by wkentaro** [comment]
it has already been fixed via
https://github.com/jsk-ros-pkg/jsk_recognition/commit/56ef006ccd7c42cce08022a73bebd0723340279f

**Comment by k-okada** [comment]
oh, i see

**Pull request merged**